### ISSUE TITLE
🐞 [Bug]: iOS 18 - no channel messages

### ISSUE THREAD

**Issue by rinchen** [meshtastic/Meshtastic-Apple#818]
### Firmware Version

2.3.6

### What did you do?

Upgraded to public beta of iOS 18 and the channel message area is blank. 
![IMG_4362F4C179D3-1](https://github.com/user-attachments/assets/ce5a8e66-1ce4-4ac9-98df-2cf333adf3c2)

### Expected Behavior

The ability to see channel messages like in iOS 17th

### Current Behavior

White screen

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by garthvh** [comment]
There is no iOS 18 build at all yet, please report any bugs you find with the beta to apple.

**Comment by Lolagatorade** [comment]
Yeah would be a pretty critical fix considering iOS 18 release is right around the corner.

**Comment by Lolagatorade** [comment]
Direct messages work no issue running in latest iOS 18 dev beta. ![image](https://github.com/user-attachments/assets/401cad50-08b0-439b-a754-ecae6badfd2b)
![image](https://github.com/user-attachments/assets/e9ad7ee6-0a78-4bfb-bba3-f09f5b859992)

**Comment by garthvh** [comment]
Keep reporting bugs to Apple

**Comment by djh1997** [comment]
> Direct messages work no issue running in latest iOS 18 dev beta. ![image](https://private-user-images.githubusercontent.com/42100725/352910720-401cad50-08b0-439b-a754-ecae6badfd2b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NDI2NDgsIm5iZiI6MTcyMjQ0MjM0OCwicGF0aCI6Ii80MjEwMDcyNS8zNTI5MTA3MjAtNDAxY2FkNTAtMDhiMC00MzliLWE3NTQtZWNhZTZiYWRmZDJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDE2MTIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzMmU1NWE5YmFlNjJiNjEzZmEzYzhiYzc4MDE4NTE5OGQ0YjEzYzY2MWQ1ZDE1NDhiOWI3ZWIyOTBmOTgyYTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UXnvbF6d2BaM0Q9ZMh5ID7uNClIPmCggFsMVqNQddbk) ![image](https://private-user-images.githubusercontent.com/42100725/352910755-e9ad7ee6-0a78-4bfb-bba3-f09f5b859992.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NDI2NDgsIm5iZiI6MTcyMjQ0MjM0OCwicGF0aCI6Ii80MjEwMDcyNS8zNTI5MTA3NTUtZTlhZDdlZTYtMGE3OC00YmZiLWJiYTMtZjA5ZjViODU5OTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDE2MTIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4YWRiYjA4NDM4M2I5YzhhYWQwMjBmOWM0OGQ4YjVjOTY3MGE2ZjM5NGE4NTA3NDc4MjIxZDdlMjdlOTM0NGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.42u49ON-87DPCnq-GekzBPX5A3gSPA_oKlC2i702nFI)

Definitely seems like an app issue as opposed to an ios issue

**Comment by Lolagatorade** [comment]
> > Direct messages work no issue running in latest iOS 18 dev beta. ![image](https://private-user-images.githubusercontent.com/42100725/352910720-401cad50-08b0-439b-a754-ecae6badfd2b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NDI2NDgsIm5iZiI6MTcyMjQ0MjM0OCwicGF0aCI6Ii80MjEwMDcyNS8zNTI5MTA3MjAtNDAxY2FkNTAtMDhiMC00MzliLWE3NTQtZWNhZTZiYWRmZDJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDE2MTIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzMmU1NWE5YmFlNjJiNjEzZmEzYzhiYzc4MDE4NTE5OGQ0YjEzYzY2MWQ1ZDE1NDhiOWI3ZWIyOTBmOTgyYTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UXnvbF6d2BaM0Q9ZMh5ID7uNClIPmCggFsMVqNQddbk) ![image](https://private-user-images.githubusercontent.com/42100725/352910755-e9ad7ee6-0a78-4bfb-bba3-f09f5b859992.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0NDI2NDgsIm5iZiI6MTcyMjQ0MjM0OCwicGF0aCI6Ii80MjEwMDcyNS8zNTI5MTA3NTUtZTlhZDdlZTYtMGE3OC00YmZiLWJiYTMtZjA5ZjViODU5OTkyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDE2MTIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4YWRiYjA4NDM4M2I5YzhhYWQwMjBmOWM0OGQ4YjVjOTY3MGE2ZjM5NGE4NTA3NDc4MjIxZDdlMjdlOTM0NGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.42u49ON-87DPCnq-GekzBPX5A3gSPA_oKlC2i702nFI)
> 
> 
> 
> Definitely seems like an app issue as opposed to an ios issue 

Exactly what I was thinking but if that's where the line is drawn and the conclusion that is made, I'm just gonna stay quiet I guess. 

Because it makes no sense for it to work in one version of iOS and not in the other it just seems like API update that broke the app and the application needs to be updated to adapt to that 

Apple just makes the platform it's up to the developers to accommodate to any updates and changes.

**Comment by garthvh** [comment]
Fetched properties are not working on iOS18, pretty clearly an OS issue.

**Comment by Lolagatorade** [comment]
> Fetched properties are not working on iOS18, pretty clearly an OS issue.

Is that specific to ios18?
Because on ipados![image](https://github.com/user-attachments/assets/a69dfbfb-03c4-49ef-8e11-e3fb8c428a21)18 it's working just fine now with my testing.
Seems to be a GUI/UX issue

Look at the end of the day you're the maintainer and you're the expert on this issue. I'm just saying that it seems like it's more than just an operating system issue given that has already been 4 beta updates and it is still ongoing. 

Attached you will see two different messages being sent from an iPad to the long fast channel successfully on two different Meshtastic devices. It is also received on my iPhone, but I am unable to open and reply. Nothing shows. But it still shows up on the little notification badge.

Maybe it's too zoomed in where the messages aren't seen because I do notice. There are some strange scaling issues on the iPad who knows just a suggestio ![image](https://github.com/user-attachments/assets/d9c26c74-5701-4145-bd4e-4edf1413c371)suggestion.



![image](https://github.com/user-attachments/assets/7071c7b8-3f6b-4da2-859d-4d01b3851e40)
![image](https://github.com/user-attachments/assets/ba26d218-8917-47d3-b060-8bbb461154b9)
 ![image](https://github.com/user-attachments/assets/af7fe1f7-711e-4d53-a214-9a7e29aa9674)![image](https://github.com/user-attachments/assets/615253eb-815d-4644-93f0-8cd398538e14)n![image](https://github.com/user-attachments/assets/9ab859c3-77b4-4bed-8877-6ccec14def05)

**Comment by djh1997** [comment]
looks like its being fixed in [this pull request](https://github.com/meshtastic/Meshtastic-Apple/pull/825)

**Comment by djh1997** [comment]
For me some times when I use the landscape layout I can get it to load.

**Comment by Lolagatorade** [comment]
> For me some times when I use the landscape layout I can get it to load.

Yeah there's something going on with aspect ratios and scaling because even in iPad when using different window modes it either works or doesent 
![image](https://github.com/user-attachments/assets/c82ec873-b0dc-4039-90a2-10177d3acebb)

![image](https://github.com/user-attachments/assets/89b263d8-8d21-4caf-82ea-b078407f8ece)

**Comment by garthvh** [comment]
Closed by https://github.com/meshtastic/Meshtastic-Apple/pull/825

**Comment by garthvh** [comment]
I keep closing these issues because we have known from the start what was broken.
